keras/layers/__init__.py

```diff
@@ -49,6 +153,11 @@ def deserialize(config, custom_objects=None):
     globs = globals()  # All layers.
     globs['Model'] = models.Model
     globs['Sequential'] = models.Sequential
+    from keras.engine.input_layer import InputLayer
+    globs['InputLayer'] = InputLayer
+    from .recurrent import StackedRNNCells
```

Comment:
You can put this in `layers`, it's not documented but safe to use (the reason it's not document is that you would just it via `RNN(list_of_cells)`)

Reply:
Ok I believe this should be documented at some point.